README.md

```diff
@@ -1,4 +1,4 @@
-# Python_and_the_Web
+# Python_and_the_Web_an_Amazing_Project
```

Comment:
Change this back to original

Reply:
Add this Line: **An amazing project for Python lovers**, after the description of project and before the Note.